src/index.html

```diff
@@ -43,6 +43,7 @@
               <li data-ng-class="{active: $state.includes('news')}"><a data-ui-sref="news.list">News</a></li>
               <li data-ng-show="canManagePermissions" data-ng-class="{active: $state.includes('admin.permissions')}"><a data-ui-sref="admin">Permissions</a></li>
               <li data-ng-class="{active: $state.includes('about')}"><a data-ui-sref="about">About</a></li>
+              <li><a href="/wiki">Wiki</a></li>
```

Comment:
Can you make this link like the others above, so that it will be styled as active when it is active? At least, until we've got the external wiki set up.